components/browser/icons/build.gradle

```diff
@@ -25,6 +25,10 @@ android {
         exclude 'META-INF/proguard/androidx-annotations.pro'
     }
 
+    aaptOptions {
+        ignoreAssetsPattern "manifest.json.template"
```

Comment:
Does it make sense to use something like `manifest.template.json` so that the editor can guess from the file extension that it is JSON?

Reply:
We didn't want this file to be accidentally used for anything by the extension code, since we currently won't be stripping this file from the apk; I'm not sure if that's an actual problem or just a healthy dose of paranoia, but it does also make sense to make the IDE experience a bit nicer.

Reply:
I thought this aapt config is for stripping it from the apk?

Reply:
> Does it make sense to use something like manifest.template.json

Yeah, good point, since it is valid JSON even with the template/placeholder, let's do this. Updated.

> I thought this aapt config is for stripping it from the apk?

Yes, stripping it from the APK, to be paranoid ... making sure it doesn't get used by mistake somehow and we have an extension running with version "${version}". :).